eos-core-depends

```diff
@@ -300,6 +300,8 @@ xserver-xorg-input-synaptics
 # Generic ARM X driver
 xserver-xorg-video-armsoc [armhf]
 # Some of these are really specific to x86, but just keep them off of arm for now
+xserver-xorg-video-ati [!armhf]
```

Comment:
@dsd I don't think you want this. ati is a wrapper driver that depends on radeon (which we have below) in addition to the old xserver-xorg-video-r128 and xserver-xorg-video-mach64, which we don't have. I'm pretty sure this will just break the ostree build.